sqlglot/optimizer/merge_subqueries.py

```diff
@@ -38,7 +38,7 @@ def merge_subqueries(expression, leave_tables_isolated=False):
 
 
 # If a derived table has these Select args, it can't be merged
-UNMERGABLE_ARGS = set(exp.Select.arg_types) - {
+unmergeable_ARGS = set(exp.Select.arg_types) - {
```

Comment:
why'd you chaneg this?

Reply:
Looks like a bad find/replace. Will fix.